ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -343,16 +346,18 @@ import { DraggableListBehavior } from '../nuxeo-draggable-list-behavior.js';
                       ></nuxeo-data-table-row-detail>
                     </template>
                   </dom-if>
-                  <nuxeo-data-table-row-actions
-                    index="[[index]]"
-                    editable="[[editable]]"
-                    orderable="[[orderable]]"
-                    template="[[rowForm]]"
-                    item="[[item]]"
-                    size="[[items.length]]"
-                    table="[[_this]]"
-                  >
-                  </nuxeo-data-table-row-actions>
+                  <div style$="[[_computeActionsStyle(editable, orderable)]]">
```

Comment:
Should it depend also on `items.length`

Reply:
We don't need it here. We only need it on the header row to observe if the scrollbar is present.